### PR TITLE
feat: add `recall`, nodes-stat, `app` command

### PR DESCRIPTION
**Pull request by Shuo-Jia**
This pr add `recall` and `app` command, and **it need be noticed**:

- split `nodes -d` into `nodes-stat` to show the per-counter stat
- `nodes` support `nodes -a temp` to show one app replica count to replace the origin `app temp` . And now `app temp` only show the partition configruation

### CODE REVIEW

File: executor/list_nodes.go [hunk 1]
```diff
@@ -11,207 +11,85 @@ import (
...
-	// nodes -q
-	GetQPS    int64
-	MgetQPS   int64
-	PutQPS    int64
-	MputQPS   int64
-	GetBytes  int64
-	MgetBytes int64
-	PutBytes  int64
-	MputBytes int64
+type NodeInfo struct {
```
Comment (neverchanje) [inline executor/list_nodes.go:14]:
```suggestion
type nodeInfo struct {
```

File: executor/nodes_stat.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	"admin-cli/helper"
+	"context"
+	"fmt"
+	"github.com/XiaoMi/pegasus-go-client/idl/admin"
+	"github.com/olekukonko/tablewriter"
+	"strconv"
+	"time"
+)
+
+type NodeStat struct {
```
Comment (neverchanje) [inline executor/nodes_stat.go:13]:
```suggestion
type nodeStat struct {
```

File: executor/nodes_stat.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+	DiskTotalMB  int64
+	DiskUsedMb   int64
+	DiskRatio    int64
+	MemUsedMB    int64
+	BlockCacheMB int64
+	MemTableMB   int64
+	MemIdxMB     int64
+}
+
+func NodesStat(client *Client, detail bool) error {
```
Comment (neverchanje) [inline executor/nodes_stat.go:34]:
```suggestion
func ShowNodesStat(client *Client, detail bool) error {
```

Do not give the same to both struct and function. This rule is applicable to every programming language.

File: executor/table_partition.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	"context"
+	"fmt"
+	"github.com/olekukonko/tablewriter"
+	"strconv"
+	"time"
+)
+
+type PartitionConfigration struct {
+	Pidx               int32
+	Ballot             int64
```
Comment (neverchanje) [inline executor/table_partition.go:14]: Do we really need to display the ballot? Have you used it once?

File: cmd/table_partition.go [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import (
+	"admin-cli/executor"
+	"admin-cli/shell"
+	"fmt"
+	"github.com/desertbit/grumble"
+)
+
+func init() {
+	shell.AddCommand(&grumble.Command{
+		Name:      "app",
```
[the remark this answers is not in the document]
Comment (Shuo-Jia) [reply cmd/table_partition.go:12]: Yes, sometime we need check node migrate status to see which replica is missing

File: executor/nodes_stat.go (hunk 3 above)
Comment (Shuo-Jia) [reply executor/nodes_stat.go:34]: Right, the struct is `nodeStat`, but function is `nodesStat`

File: executor/table_partition.go (hunk 4 above)
Comment (Shuo-Jia) [reply executor/table_partition.go:14]: Well, I just think may be useful, actually I havent used it

**Review by neverchanje — APPROVED** [review]

**Pull request merged**